server/test/write.go

```diff
@@ -43,6 +43,10 @@ var writeCommandTests = []writeCommandTest{
 		_name: "ExecuteWithSameTupleInWritesReturnsError",
 		// state
 		typeDefinitions: []*openfgapb.TypeDefinition{
+			{
+				Type:      "user",
```

Comment:
Shouldn't all of the prior tests still work without adding an explicit user type? I think we should be adding tests asserting new behavior for writes to models under the new schema_version 1.1. The existing tests are under schema_version 1.0 and should remain as is, yeah?

Reply:
You're right! When I wrote the original commit i didn't have the schema version available. 

Added it in recent commits and went with option b. The existing tests target 1.0, the ones i added target 1.1.